### PR TITLE
improve build / run instructions

### PR DESCRIPTION
**Pull request by obestwalter**
Trying it out on Arch Linux I noticed that the install step is missing, before building.

I also fixed a minor formatting problem.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # deltachat-desktop
 
-> Desktop Application for [delta.chat](https://delta.chat)
+**Desktop Application for [delta.chat](https://delta.chat)**
```
Comment (ralphtheninja) [inline README.md]: Why this change?

Comment (obestwalter) [reply README.md]: I assumed that this was an oversight (and to me it looks wrong). 

`>` is meant for quotes but that sentence isn't, so I replaced it with another form of emphasis that doesn't look like somebody accidentally formatted it as a quote.

Comment (ralphtheninja) [reply README.md]: It's quite common to use `>` for short descriptions in README.

Comment (ralphtheninja) [reply README.md]: But I'll let your change pass to keep _everyone_ happy but me.
